appveyor.yml

```diff
@@ -94,6 +94,12 @@ environment:
         COMPILER_PATH: "C:\\MinGW\\bin"
         MSYS2_ARG_CONV_EXCL: "/*"
         BUILD_OPT: -k
+      - APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2015"
+        BUILD_SYSTEM: winbuild
+        OPENSSL: ON
+        WINSSL: OFF
+        COMPILER_PATH: "C:\\MinGW\\bin"
```

Comment:
It can probably use some further trimming, yes. I wasn't quite sure what this would end up using. I'm still struggling with the more basic parts of just making the build kick off...